src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const sum = (application1) => {
+    const addition = (application2) => {
+      count++;
+
+      if (count > 3 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return application1 + application2;
+    };
+
+    return addition;
+  };
+
+  return sum;
```

Comment:
you can return both of these functions right away, without creating variables for them